src/mcp3564.c

```diff
@@ -163,21 +163,67 @@ void MCP3561_Reset(SPI_HandleTypeDef *hspi){
 
 /**
  * @brief read 24 Bit left justified ADC register
- * @todo  how to read from other data formats?
+ * @todo  obsolete?

```

Comment:
i agree, but good thing to leave the function still here. to be "backwards compatible"